cypress/integration/transforms_spec.js

```diff
@@ -0,0 +1,247 @@
+
+        // Get description input box
+        cy.get(`textarea[data-test-subj="description"]`).focus().type("some description");
+
+        // Enter source index
+        cy.get(`div[data-test-subj="sourceIndexCombobox"]`)
+          .find(`input[data-test-subj="comboBoxSearchInput"]`)
+          .focus()
+          .type("opensearch_dashboards_sample_data_ecommerce{enter}");
+
+        // Enter target index
+        cy.get(`div[data-test-subj="targetIndexCombobox"]`)
+          .find(`input[data-test-subj="comboBoxSearchInput"]`)
+          .focus()
+          .type("target_index{enter}");
+
+        // Click the next button
+        cy.get("button").contains("Next").click({ force: true });
+
+        // Confirm that we got to step 2 of creation page
+        cy.contains("Select fields to transform");
+
+        // Setup Group and aggregation
+        cy.wait(2000);
+
+        /* Finds the correct header text, then navigates up the structure, then
+         * down to the next div, which contains the button. Then navigates down
+         * the div to the clickable button and clicks it.
+         */
+        cy.contains("category.keyword").parent().parent().parent().next()
+          .children().first().children().first().children().first()
+          .click({ force: true });
+
+        cy.contains("Group by terms").click({ force: true });
+
+        // Confirm group was added
+        cy.contains("category.keyword_terms");
+
+        // Add aggregable field
+        cy.contains("50 columns hidden").click({ force: true });
+        cy.contains("taxless_total_price").click({ force: true });
+        // Click out of the window
+        cy.contains("Select fields to transform").click({ force: true });
+
+        /* Finds the correct header text, then navigates up the structure, then
+         * down to the next div, which contains the button. Then navigates down
+         * the div to the clickable button and clicks it.
+         */
+        cy.contains("taxless_total_price").parent().parent().parent().next()
+          .children().first().children().first().children().first()
+          .click({ force: true });
+        cy.contains("Aggregate by avg").click({ force: true });
+
+        // Confirm agg was added
+        cy.contains("avg_taxless_total_price");
+
+        // Click the next button
+        cy.get("button").contains("Next").click({ force: true });
+
+        // Confirm that we got to step 3 of creation page
+        cy.contains("Job enabled by default");
+
+        // Click the next button
+        cy.get("button").contains("Next").click({ force: true });
+
+        // Confirm that we got to step 4 of creation page
+        cy.contains("Review and create");
+
+        // Click the create button
+        cy.get("button").contains("Create").click({ force: true });
+
+        // Verify that sample data is add by checking toast notification
+        cy.contains(`Transform job "${TRANSFORM_ID}" successfully created.`);
+      });
```

Comment:
Do you also want to add a check to confirm that we got routed back to the table page and that the table correctly shows a new transform was added?